Sources/StreamChatUI/CommonViews/QuotedChatMessageView/QuotedChatMessageView.swift

```diff
@@ -124,6 +124,10 @@ open class _QuotedChatMessageView<ExtraData: ExtraDataTypes>: _View, ThemeProvid
         guard let message = content?.message else { return }
         guard let avatarAlignment = content?.avatarAlignment else { return }
 
+        contentContainerView.backgroundColor = message.linkAttachments.isEmpty
+            ? appearance.colorPalette.background
+            : appearance.colorPalette.highlightedAccentBackground1
+
```

Comment:
This should ideally come from the injector we don't have such a thing here 🤔 Let's leave it as is and this about a better solution later...

Reply:
The `QuotedChatMessageView` is independent of the injectors since it can be used outside of it (Composer). For me, it makes sense that the color is defined here since the background color is impacted by the inner content of the quoted message, not by where the quoted message is (Although this completely possible to do as well, if the customer's logic is different).